src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+quarkus.kogito.devservices.enabled=false
+quarkus.devservices.enabled=false
```

Comment:
I'd suggest to add the following in order for native images to work properly:
`quarkus.native.resources.includes=**swidtag`

reference: https://github.ibm.com/dba/bamoe-issues/issues/76

Reply:
great catch @martinweiler !